config/nginx/conf.d/cantusdb.conf.production

```diff
@@ -27,8 +30,9 @@ server {
 }
 
 server {
-
     listen 443 default_server http2 ssl;
+
+    server_name cantusdatabase.org www.cantusdatabase.org;
```

Comment:
As I mentioned to @dchiller, you don't really want to do this, since it will consider `cantusdatabase.org` and `www.cantusdatabase.org` to both be valid addresses. You should set up a different server and redirect any traffic coming from `www.cantusdatabase.org` to the canonical `cantusdatabase.org` instead.

Reply:
You did say this...and it is so in the ansible set-up...my bad for not catching here!